commcare_connect/templates/opportunity/invoice_download.html

```
@@ -1,87 +1,170 @@
-{% load static i18n %}
+{% load static i18n humanize %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)